src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -72,3 +103,29 @@ instance CustomErrorHasDoc "fROZEN_TOKEN_NOT_TRANSFERABLE" where
   customErrClass = ErrClassActionException
   customErrDocMdCause =
     "The sender tries to transfer frozen token"
+
+-- Errors
+
+type instance ErrorArg "nO_PENDING_ADMINISTRATOR_SET" = ()
+
+instance CustomErrorHasDoc "nO_PENDING_ADMINISTRATOR_SET" where
+  customErrClass = ErrClassActionException
+  customErrDocMdCause = "Received an `accept_ownership` call when no pending owner was set"
+  customErrArgumentSemantics = Nothing
+
+type instance ErrorArg "nOT_PENDING_ADMINISTRATOR" = ()
+
+instance CustomErrorHasDoc "nOT_PENDING_ADMINISTRATOR" where
+  customErrClass = ErrClassActionException
+  customErrDocMdCause =
+    "Received an `accept_ownership` from an address other than what is in the pending owner field"
+  customErrArgumentSemantics = Nothing
+
+type instance ErrorArg "nOT_ADMINISTRATOR" = ()
+
+instance CustomErrorHasDoc "nOT_ADMINISTRATOR" where
+  customErrClass = ErrClassActionException
+  customErrDocMdCause =
+    "Received an operation that require administrative privileges\
+    \ from an address that is not the current administrator"
+  customErrArgumentSemantics = Nothing
```

Comment:
Well, it's `Nothing` by default, I am not sure there is benefit in specifying `Nothing` explicitly, but I don't mind.

Reply:
Changed